budgetportal/webflow/urls.py

```diff
@@ -12,4 +12,9 @@
         views.provincial_infrastructure_project_detail,
         name="provincial-infra-project-detail",
     ),
+    url(
+        r"^infrastructure-projects/provincial/search/$",
+        views.ProvInfraProjectView.as_view(),
+        name="search-provincial-infrastructure-projects",
```

Comment:
let's call this `provincial-infrastructure-project-api`

Reply:
name changed.

https://github.com/vulekamali/datamanager/pull/190/commits/506e3913fdaee7ec173ecd6042251c18ebf9cba4